src/tbp/monty/simulators/habitat/actuator.py

```diff
@@ -66,170 +65,119 @@ def action_name(self, action: Action) -> str:
         """
         return f"{action.agent_id}.{action.name}"
 
-    def actuate(self, action: Action, parameterizer: Type[HabitatParameterizer]):
```

Comment:
What's the reason we can just remove the general `actuate` function here?

Reply:
Yes to what @jeremyshoemaker said. This method was only called by `actuate_*` methods. However, it violated some design principles, so it got reworked into `to_habitat`, which is also only called by the `actuate_*` methods.